packages/components/src/FilterSelect/subcomponents/ListBoxSection/ListBoxSection.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { useListBoxSection } from "@react-aria/listbox"
+import { Heading } from "@kaizen/typography"
+import { SelectOption, SelectOptionGroupNode } from "../../types"
+import { Option } from "../Option/Option"
+import styles from "./ListBoxSection.module.scss"
+
+export interface ListBoxSectionProps<Option extends SelectOption> {
+  section: SelectOptionGroupNode<Option>
+}
+
+export const ListBoxSection = <Option extends SelectOption = SelectOption>({
+  section,
+}: ListBoxSectionProps<Option>): JSX.Element => {
+  const { itemProps, headingProps, groupProps } = useListBoxSection({
+    heading: section.rendered,
+    "aria-label": section["aria-label"],
+  })
+
+  return (
+    <>
+      <li {...itemProps} className={styles.sectionWrapper}>
+        {section.rendered && (
+          <Heading
+            variant="heading-6"
```

Comment:
@dougmacknz should this be a `h6`? This was copied from `Select` but just wanted to double check.

Reply:
Hm, this actually ends up with `aria-hidden=true`.
<img width="687" alt="image" src="https://user-images.githubusercontent.com/1811583/228105415-01aab1d4-88e3-4fe9-a123-3fe95a594e98.png">

The react-aria docs have these as spans https://react-spectrum.adobe.com/react-aria/useListBox.html#sections

I think we should switch this to a span just to make it clear we don't want a header here.

Reply:
I reckon this was just another case of forgetting to use the `tag` prop. It happens all the time, even in our team.